artificial-scrum-master/apps/artificial.scrum.master.frontend/src/app/features/tasks/components/edit-task-details/edit-task-details.component.ts

```diff
@@ -26,80 +33,53 @@ export class EditTaskDetailsComponent {
   private readonly storyTaskEditService = inject(StoryTaskEditService);
   private readonly toastService = inject(ToastService);
 
-  details = model.required<TaskDetails | null>();
+  @Input() editorStateServiceService: EditorStateServiceService;
```

Comment:
użyj signala

Reply:
musisz to przemyśleć - zrób injecty - serwisów inputem nie przekazujemy

Reply:
*przy zamknięciu popUpa - czyszczenie serwisu